### ISSUE TITLE
SDL_Error: Joystick hasn't been opened yet.

### ISSUE THREAD

**Issue by FusionLord** [electronstudio/sdl2gdx#14]
Afterglow: Controller for PS3
Reads controller USB receiver but doesn't initialize once it connects.

https://pastebin.com/5ttALVhJ

**Comment by electronstudio** [comment]
Do other controllers work, i.e. this problem is specific to Afterglow?

**Comment by FusionLord** [comment]
Yes other controllers work fine, xinput ds4, seems specific to the Afterglow. may be specific to controllers that have wireless usb receivers, Only have the one to test.

**Comment by electronstudio** [comment]
Sounds like an SDL issue.  Does it work other SDL apps, e.g. the sdljoystick.c example that comes with SDL?

**Comment by FusionLord** [comment]
How do i run the test? I have visual studio with SDL loaded into it.

**Comment by electronstudio** [comment]
sorry i dont have visual studio so i cant provide step by step instructions.

sdl2gdx has updated to SDL 2.0.12 so i suggest trying it again now and see if it works.
